### ISSUE TITLE
Does the order of Java api runner.feed(#1).feed(#2).feed(#3)... matters?

### ISSUE THREAD

**Issue by sxqqslf** [tensorflow/tensorflow#20837]
Hi, guys, I have a quesion about the Tensorflow Java api.
When I feed the data to the model, the feed order matters ?
For example, runner.feed(#1).feed(#2).feed(#3)... is equal to runner.feed(#2).feed(#1).feed(#3) ? 
Besides, recently, I run some response generation work from github(https://github.com/tuxchow/ecm), which use the old seq2seq code (implement with buckets), and i use the python to train the model, java to predict, as a result, I feed the test sentence word by word into the model, e.x. runner.feed(word1).feed(word2).feed(word3)... , However, I found that except the first word's logits is correct,  from the second word it's wrong, why is that?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by jdduke** [comment]
The order shouldn't matter as long as each call to `runner.feed()` is provided a unique `inputName`. As for why other outputs would be wrong, it's hard to say without access to the model. Any chance you can share the model in question?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?
